src/scenes/details/Details.js

```diff
@@ -14,14 +14,33 @@ class Details extends Component {
     headerTitleStyle: homeStyles.text
   };
 
+  markAsDone = (id) => {
+    const { navigation } = this.props;
+    navigation.navigate('Home', {
+      idTask: id,
+    });
+  }
+
   render() {
     const { navigation } = this.props;
-    const task = navigation.getParam("task");
+    const task = navigation.getParam('task');
     return (
       <View style={homeStyles.listContainer}>
         <View style={detailStyles.container}>
+          <Text style={detailStyles.notDone}>
+            {task.done ? 'Done' : 'Not Done'}
```

Comment:
Just a comment maybe you can add this to strings file like you did with the colors.